cmd/code/push.go

```diff
@@ -62,22 +63,33 @@ func init() {
 			if err := addDatasetIfNotExisted(ctx, datasetConfig); err != nil {
 				return err
 			}
+			saveDatasetVersionParams := *tensorleapapi.NewSaveDatasetVersionParams(
+				datasetConfig.DatasetId,
+				uploadUrl,
+				datasetConfig.EntryFile,
+			)
+
+			if len(secretId) > 0 {
```

Comment:
Is >0 the only requirement or do we want to check if the secret exists (and indicate an error if it doesnt) or that the id is a guid or something more specific

Reply:
@YuvalFeldman The `secretId` is string and string in go can't be `nill`

Reply:
we can check if the id is exists but I expect that node will raise an NotFoundError if the secret not exists